config/config_test.go

```diff
@@ -493,3 +427,68 @@ func TestLocal_DNSBootstrap(t *testing.T) {
 		})
 	}
 }
+
+func TestLocalStructTags(t *testing.T) {
+	localType := reflect.TypeOf(Local{})
+
+	versionField, ok := localType.FieldByName("Version")
+	require.True(t, true, ok)
+	ver := 0
+	versionTags := []string{}
+	for {
+		_, has := versionField.Tag.Lookup(fmt.Sprintf("version[%d]", ver))
+		if !has {
+			break
+		}
+		versionTags = append(versionTags, fmt.Sprintf("version[%d]", ver))
+		ver++
+	}
+
+	for fieldNum := 0; fieldNum < localType.NumField(); fieldNum++ {
+		field := localType.Field(fieldNum)
+		if field.Tag == "" {
+			require.Failf(t, "Field is missing versioning information", "Field Name: %s", field.Name)
+		}
+		if field.Name == "Version" {
```

Comment:
nit: add a comment `Version` is tested in `TestLocalVersionField`

Reply:
sure.